pages/_app.tsx

```diff
@@ -10,8 +10,7 @@ export default function App({ Component, pageProps }: AppProps) {
       <meta content="A Protocol for Decentralized Development" name="description" />
       <meta content="A Protocol for Decentralized Development" property="og:description" />
       <meta content="Bepro.network - Web 3.0 codebase protocol" property="twitter:title" />
```

Comment:
I think we forgot one issue, replace "Bepro.network - ..." with "Bepro.network | ..."